Source/WebCore/dom/WindowOrWorkerGlobalScopeTrustedTypes.cpp

```diff
@@ -92,20 +92,18 @@ class WorkerGlobalScopeTrustedTypes : public Supplement<WorkerGlobalScope> {
 private:
     static ASCIILiteral supplementName() { return "WorkerGlobalScopeTrustedTypes"_s; }
 
-    WorkerGlobalScope& m_scope;
```

Comment:
@lukewarlow : is it ok to remove `m_scope`?
Or do you plan to make use of it in future patchsets?

Reply:
https://github.com/WebKit/WebKit/pull/23999 - I make use of it here. But feel free to remove it and I can readd in that patch when I come to use.

Reply:
Ok, I'll close this PR then, no need to pollute the code history if the build issue is only transient.
Thanks for your quick feedback!